.eslintrc.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+  "root": true,
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module"
+  },
+  "env": {
+    "es6": true,
+    "node": true,
+    "browser": true
+  },
+  "plugins": [
+    "implicit-dependencies"
+  ],
+  "extends": [
+    "eslint:recommended",
```

Comment:
Could we switch from the default eslint rules to Airbnb?

Reply:
Hmm, we would need to further refactor a bunch of stuff to make everything pass the airbnb standards. I think we should create a new ticket if we want to tackle that later.

We’ll need to update the current rules, however, to prefer single quotes in JS.

Reply:
This is our current rule set for webstarter js projects. We should probably update to match. https://github.com/forumone/generator-web-starter/blob/master/.eslintrc.js

Reply:
This also incorporates typescript. Which I dont think we need to do yet.

Reply:
My single quote prettier errors were due to me not having `.prettierrc` in the Dockerfile.